Sources/KituraContracts/ClousureAliases.swift

```diff
@@ -18,49 +18,73 @@
 
 /**
 The `ResultClosure` is used by other `Codable` aliases when responding with only a `RequestError` is needed
-*/
+ 
+ ````swift
+ // The following two typealias make use of ResultClosure
+ public typealias NonCodableClosure = (@escaping ResultClosure) -> Void
+ 
+ public typealias IdentifierNonCodableClosure<Id: Identifier> = (Id, @escaping ResultClosure) -> Void
+ ````
+ */
 public typealias ResultClosure = (RequestError?) -> Void
 
 /**
 The `CodableResultClosure` is used by other `Codable` aliases when responding with an object which conforms to Codable or an `RequestError` is needed.
-*/
```

Comment:
a `RequestError` not an `RequestError`

Reply:
perhaps add the word either so it reads "Used by other `Codable` aliases when either, responding with an object which conforms to `Codable`, or a `RequestError` is needed."